### PR TITLE
Feature/init network stuff

### PR DESCRIPTION
**Pull request by barweiss**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM alpine
+
+RUN apk add --no-cache iptables
+
+COPY init.sh /usr/local/bin/
```
Comment (val06) [inline Dockerfile:5]: I think it is better to copy scripts to a dedicated folder and change the WD to that folder (similar to [sniffer-docker](https://github.com/seek-ret/sniffer-docker/blob/master/Dockerfile) docker file)

**Review by val06 — CHANGES_REQUESTED** [review]
cr-ed

File: Dockerfile (hunk 1 above)
Comment (barweiss) [reply Dockerfile:5]: Actually they copy the scripts to `/bin`

Comment (val06) [reply Dockerfile:5]: 😕

**Pull request merged**